HelpSource/Classes/LatoocarfianL.schelp

```diff
@@ -63,3 +63,24 @@ code::
 ) * 0.2 }.play(s);
 )
 ::
+
+code::
+// randomly modulate all params
+// randomly modulate all params
+(
+~thing = 
```

Comment:
thanks a lot for all the nice improvements!

should we briefly discuss the helpfile style regarding the `~something`s ? It would be good if we all agreed on their use of non-use in the ugen helpfiles.